backend/tracim_backend/lib/utils/utils.py

```diff
@@ -541,3 +543,18 @@ def validate_data(self, params: Dict[str, Any], json_schema: Dict[str, Any]):
                     "> ".join([str(item) for item in exc.absolute_path]), exc.message
                 )
             ) from exc
+
+
+def date_as_lang(datetime_obj: datetime, locale: str = None, default_locale: str = None):
```

Comment:
Missing type annotation for returned value.

Reply:
And `Optional` for the strings.

Reply:
done.